app/templating/schema_context.py

```diff
@@ -46,27 +47,27 @@ def _build_answers(answer_store, answer_ids_on_path):
     return answers
 
 
-def _build_exercise(metadata):
-    return {
-        'start_date': metadata['ref_p_start_date'],
-        'end_date': metadata['ref_p_end_date'],
-        'period_str': json_and_html_safe(metadata['period_str']),
-        'employment_date': metadata['employment_date'],
-        'return_by': metadata['return_by'],
-        'region_code': json_and_html_safe(metadata['region_code']),
-    }
+def build_schema_metadata(metadata):
 
+    schema_metadata = load_schema_from_params(metadata['eq_id'], metadata['form_type']).json['metadata']
```

Comment:
You might as well just use `g.schema` rather than `load_schema_from_params(metadata['eq_id'], metadata['form_type'])`.